xml/System.Runtime.InteropServices/DllImportSearchPath.xml

```diff
@@ -137,7 +137,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
+        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. When used in NativeAOT, the application's directory is considered the "assembly directory" and searched. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
```

Comment:
> This value is used by .NET on Windows before the paths are passed to the Win32

I am not sure what this is trying to say. What is "this value" that it refers to and how it is used?

Reply:
I _think_ the "this value" is referring to the enumeration `AssemblyDirectory` and this is saying the value isn't passed to the OS loader because the runtime performs the lookup for the "assembly directory". At least, that is how I read it. All of the enum value descriptions here should probably be updated honestly.

Reply:
I'm going to remove that statement as I think it is noise and expressing implementation details.

Reply:
I will capture the rest of @gewarren's changes though.